server/graphql/v2/query/collection/ExpensesCollectionQuery.ts

```diff
@@ -285,6 +285,7 @@ const ExpensesCollectionQuery = {
           [Op.or]: [
             { status: { [Op.notIn]: [expenseStatus.DRAFT, expenseStatus.SPAM] } },
             { status: expenseStatus.DRAFT, UserId: req.remoteUser.id },
+            { [Op.and]: [{ status: expenseStatus.DRAFT }, req.remoteUser.isAdminOfCollective(account)] },
```

Comment:
LOL, I think this is a first.
For consistency, I would rather assign this array to a temp variable and push it from another conditional:
```ts
const userClause = [{ status: { [Op.notIn]: [expenseStatus.DRAFT, expenseStatus.SPAM] } }]

if (req.remoteUser.isAdminOfCollective(account)) {
  userClause.push({ status: expenseStatus.DRAFT });
} else {
  userClause.push({ status: expenseStatus.DRAFT, UserId: req.remoteUser.id });
}

where[Op.and].push({ [Op.or]: userClause })
```